nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -197,8 +186,11 @@ BtNavigator::navigateToPose()
       }
     };
 
+  // Create the Behavior Tree from the XML input
+  BT::Tree tree = bt_->buildTreeFromText(xml_string_, blackboard_);
```

Comment:
So now the tree could change between navigations if the user modifies the bt xml file during run-time?